internal/cli/dump.go

```diff
@@ -134,6 +139,54 @@ func (c *dumpCliDocsCmd) run(cmd *cobra.Command, args []string) {
 	}
 }
 
+func newDumpImageDeployRefCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "image-deploy-ref REF",
+		Short: "Determine the deployed name and tag of a custom-built image",
```

Comment:
idk if this is actually clearer, take it or leave it 🤷 
```suggestion
		Short: "Determine the name and tag with which Tilt will deploy the given custom-built image",
```
(and change in `Long` description as well)

Reply:
done!